TFT/src/User/API/RRFParseACK.cpp

```diff
@@ -141,6 +190,53 @@ void ParseACKJsonParser::value(const char *value)
     case mbox_timeo:
       m291_timeo = strtod((char *)value, NULL) * 1000;
       break;
+    case resp:
+      if (strstr(value, (char *)"Steps/"))       //parse M92
+      {
+        if ((value = strstr(value, (char *)"X: ")) != NULL ) setParameter(P_STEPS_PER_MM, AXIS_INDEX_X,  atoi(value + 3));
+        if ((value = strstr(value, (char *)"Y: ")) != NULL ) setParameter(P_STEPS_PER_MM, AXIS_INDEX_Y,  atoi(value + 3));
+        if ((value = strstr(value, (char *)"Z: ")) != NULL ) setParameter(P_STEPS_PER_MM, AXIS_INDEX_Z,  atoi(value + 3));
+        if ((value = strstr(value, (char *)"E: ")) != NULL ) setParameter(P_STEPS_PER_MM, AXIS_INDEX_E0, atoi(value + 3));
+      }
+      else if ((string_start = strstr(value, (char *)"RepRapFirmware")) != NULL)    // parse M115
+      {
+        setupMachine();
+        string_end = strstr(string_start, "ELECTRONICS");
+        infoSetFirmwareName((uint8_t *)string_start, string_end-string_start);
+      }
+      else if ((string_start = strstr(value, (char *)"access point")) != NULL)    //parse M552 
+      {
+        string_end = strstr(string_start, ",");
+        string_start += 13;
+        infoSetAccessPoint((uint8_t *)string_start,  string_end-string_start);
+
+        if ((string_start = strstr(string_start, (char *)"IP address")) != NULL)
+        {
+          string_end = strstr(string_start, "\\n");
+          string_start += 11;
+          infoSetIPAddress((uint8_t *)string_start,  string_end-string_start);
+        }
+      }
+      else if ((string_start = strstr(value, (char *)"printing byte")) != NULL)       // parse M27  {"seq":21,"resp":"SD printing byte 1226/5040433\n"}
+      {
+        string_end = strstr(string_start, (char *)"/");
+        setPrintProgress(atoi(string_start + 14), atoi(string_end + 1));
+      }
+      else if (strstr(value, (char *)"Auto tuning heater") && strstr(value, (char *)"completed"))
+      {
+        pidUpdateStatus(true);
+      }
+      else if (strstr(value, (char *)"Error: M303") || (strstr(value, (char *)"Auto tune of heater") && strstr(value, (char *)"failed")))
+      {
+        pidUpdateStatus(false);
+      }
+
+      break;
+    case result:
```

Comment:
This alone is kind of dangerous, it would be good to add a new variable into RRFStatusControl: `starting_print`, set it to `true` at `mustStoreCmd("M409 K\"job.file.fileName\"\n");` and set it `false` here. This case should only run if `starting_print`.

Create new set/is functions as needed